### ISSUE TITLE
JS requestor fails without sufficient error info when run with invalid YAGNA_APPKEY

### ISSUE THREAD

**Issue by mateuszsrebrny** [golemfactory/yajsapi#220]
```
$ YAGNA_APPKEY=INCORRECT yarn js:blender -d
yarn run v1.22.5
$ node ./blender/blender.js -d
secp256k1 unavailable, reverting to browser version
Using subnet: devnet-beta.2, network: rinkeby, driver: zksync
2021-06-15 12:30:06 [yajsapi] debug: Using image repository: _girepo._tcp.dev.golem.network -> http://yacn2.dev.golem.network:8000.
2021-06-15 12:30:06 [yajsapi] error: Request failed with status code 401
2021-06-15 12:30:06 [yajsapi] debug: Executor is shutting down...
2021-06-15 12:30:06 [yajsapi] debug: Waiting for 1 computation(s)...
Done in 0.88s.
```

Full contents of the log file:
```
2021-06-15 12:33:57 [yajsapi] debug: Using image repository: _girepo._tcp.dev.golem.network -> http://yacn2.dev.golem.network:8000.
2021-06-15 12:33:57 [yajsapi] error: Request failed with status code 401
2021-06-15 12:33:57 [yajsapi] debug: Executor is shutting down...
2021-06-15 12:33:57 [yajsapi] debug: Waiting for 1 computation(s)...
```

**Comment by filipgolem** [comment]
Fixed with the following commit: f0d6fce154d98ef0da8cef144f5f767a6fe6faef.

**Comment by filipgolem** [comment]
New error message:
`Error: not authorized. Please check if YAGNA_APPKEY env variable is valid.`
